mongodb/spring-cloud-starter-stream-source-mongodb/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,3 @@
+configuration-properties.classes=org.springframework.cloud.stream.app.mongodb.source.MongoDBSourceProperties, \
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultUnlimited
```

Comment:
Indeed the `MongoProperties` should be part of the whitelist IMO, as they are a first class citizen.
I'll take a look at the rabbit and jdbc apps.